spec/decorators/zine_decorator_spec.rb

```diff
@@ -9,8 +9,23 @@
     described_class.new(zine)
   end
 
-  describe '#cover_image_tag' do
+  describe '#id' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.